.circleci/build_steps.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# PLEASE NOTE: This script has been automatically generated by conda-smithy. Any changes here
+# will be lost next time ``conda smithy rerender`` is run. If you would like to make permanent
+# changes to this script, consider a proposal to conda-smithy so that other feedstocks can also
+# benefit from the improvement.
+
+set -xeuo pipefail
+export PYTHONUNBUFFERED=1
+
+cat >~/.condarc <<CONDARC
+
+channels:
+ - conda-forge
+ - defaults
+
+conda-build:
+ root-dir: /home/conda/staged-recipes/build_artifacts
+
+show_channel_urls: true
+
+CONDARC
+
+# Copy the host recipes folder so we don't ever muck with it
+cp -r /home/conda/staged-recipes/recipes ~/conda-recipes
+cp -r /home/conda/staged-recipes/.ci_support ~/.ci_support
+
+# Find the recipes from master in this PR and remove them.
+echo "Finding recipes merged in master and removing them from the build."
+pushd /home/conda/staged-recipes/recipes > /dev/null
+git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf ~/conda-recipes/{} && echo Removing recipe: {}"
+popd > /dev/null
+
+# Unused, but needed by conda-build currently... :(
+export CONDA_NPY='19'
+
+# A lock sometimes occurs with incomplete builds. The lock file is stored in build_artifacts.
+conda clean --lock
+
+conda install --yes conda-forge-ci-setup=1.* conda-forge-pinning networkx
```

Comment:
As a cosmetic fix, adding `--quiet` here to avoid weird looking progress bars in CI logs.

xref: https://github.com/conda-forge/staged-recipes/pull/5926